charts/external-services/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0.0"
 description: Helm chart for external services. This is an optional abstraction layer for external dependencies such as redis.
 name: external-services
-version: 1.0.2
+version: 1.0.3
```

Comment:
When / why is this bump required?

Reply:
I bumped the version because there are new values as part of the external-services chart, and version bumps are required for those changed to be pushed to the public charts repository. `charts/external-services/values.yaml` and `charts/external-services/templates/service.yaml` are changed.